src/main/cms/cms.c

```diff
@@ -161,9 +161,10 @@ bool cmsDisplayPortSelect(displayPort_t *instance)
 //   13 cols x 9 rows, top row printed as a Bold Heading
 //   Needs the "smallScreen" adaptions
 
-#define CMS_MAX_ROWS 16
+#define CMS_MAX_ROWS 31
```

Comment:
shoudn't this come from the OSD grid size now?

Reply:
No, it’s the max size used to size an array, not the current canvas size.